miio/device.py

```diff
@@ -126,12 +128,16 @@ def info(self, *, skip_cache=False) -> DeviceInfo:
 
         This includes information about connected wlan network, and hardware and
         software versions.
+        This also caches the descriptors for sensors, settings and actions
+        which makes additional IO calls.
```

Comment:
```suggestion
        This method implicitly populates the internal data structures needed prior accessing `sensors()`, `settings()`, or `actions()`.
```
I don't think it is necessary to point it out here that this might do some extra requests, considering this is an external API. It is more noteworthy to mention that this needs to be called in most of the cases :-)

Reply:
Was this closed accidentally? :)